httpbp/server.go

```diff
@@ -221,9 +235,21 @@ func NewBaseplateServer(args ServerArgs) (baseplate.Server, error) {
 		return nil, err
 	}
 
+	logger, err := httpServerLogger(args.UpstreamIssueLoggingSampleRates)
+	if err != nil {
+		// Should not happen, but if it really happens, we just fallback to stdlib
+		// logger, which is not that big a deal either.
```

Comment:
Is this because we treat a nil logger as the stdlib logger somewhere? Where does that happen?

Reply:
https://cs.opensource.google/go/go/+/refs/tags/go1.19.2:src/net/http/server.go;l=3226-3232

it's also documented on [`http.Server.ErrorLog`](https://pkg.go.dev/net/http#Server.ErrorLog)